pkgs/development/python-modules/hatchling/default.nix

```
@@ -0,0 +1,58 @@
+# https://nixos.org/manual/nixpkgs/stable/#developing-with-python
+{ lib
+, buildPythonPackage
+, fetchPypi
+, intreehooks
+, pythonOlder
+, isPy27
+
+# All runtime dependencies
+, editables ? null
```

```suggestion
, editables
```
`? null` should not be used in nixpkgs